packages/cardano-services-client/src/HttpProvider.ts

```diff
@@ -2,8 +2,8 @@
 import { HttpProviderConfigPaths, Provider, ProviderError, ProviderFailure } from '@cardano-sdk/core';
 import { Logger } from 'ts-log';
 import { fromSerializableObject, toSerializableObject } from '@cardano-sdk/util';
+import { packageJson } from './version';
```

Comment:
Can we revert this to the way it worked before? It does seem like a nice change on the surface, but I think it will be a bug because when publishing sdk packages I believe we first build, then bump version 
```suggestion
```

Reply:
0b9658d17b should solve this